web/styles/left_sidebar.css

```diff
@@ -666,6 +666,57 @@ li.top_left_scheduled_messages {
     }
 }
 
+.top_left_starred_messages {
+    &.hide_starred_message_count {
+        .masked_unread_count {
+            display: block;
+            grid-area: markers-and-controls;
+            /* Adding margin-right aligns the .masked_unread_count with the rest of the masked unread counts in the channel list. */
+            margin-right: 3.2px;
```

Comment:
Is there a reason to use `3.2px` here? Comparable dots in the streams rows use `3px` (as a side note, we should really get away from using CSS to draw these dots, but in the meantime, it would be good to know where 3.2 came from.

Reply:
No, there isn't any particular reason to choose 3.2px, it just looked perfectly aligned with the other dots in the stream rows to me (by trial and error method :) ). I have also tried with 3px but it doesn't seem to get aligned with the rest of the dots (the main reason being the use of `grid-area: markers-and-controls;` for the dot's alignment in the starred message column which was not the case in stream rows).

Reply:
Got it. I'm confirming via showing grid lines that 3px is a fine value here:

![Screenshot 2024-08-23 at 2 39 09 PM](https://github.com/user-attachments/assets/bc654025-e031-4ad3-b979-3bf9a2b0ad54)

If you wanted to find the other 3px `margin-right` values for masked unreads and turn those into a CSS variable, that would be a really nice touch here.

Reply:
I have updated my PR accordingly (added a CSS variable named `--masked-unread-count-margin-right` and have used it where needed).